src/medCore/gui/medSelectorWorkspace.h

```diff
@@ -35,7 +35,7 @@ class MEDCORE_EXPORT medSelectorWorkspace : public medAbstractWorkspace
     medSelectorToolBox *selectorToolBox();
 
 protected slots:
-    virtual void onProcessSuccess(){}
+    void importProcessOutput();
```

Comment:
This should be virtual (I know the slot in ```medFilteringWorkspace```  still gets called but that's because Qt sigal-slot connection mechanism doesn't require slots to be virtual. On the other hand, if another function decides to call ```importProcessOutput``` directly, there's going to be unexpected behavior)